src/ray/common/ray_config_def.h

```diff
@@ -288,7 +288,7 @@ RAY_CONFIG(int64_t, raylet_fetch_timeout_milliseconds, 1000)
 
 /// The duration that we wait after sending a worker SIGTERM before sending
 /// the worker SIGKILL.
-RAY_CONFIG(int64_t, kill_worker_timeout_milliseconds, 100)
+RAY_CONFIG(int64_t, kill_worker_timeout_milliseconds, 5000)
```

Comment:
what's the motive for this change?

Reply:
100ms is usually too short for this sort of sigterm -> sigkill (this assumes all the clean up can be done in 100ms)